project.py

```diff
@@ -28,6 +27,11 @@ def __init__(self):
         self.width = None
         self.height = None
         self.saved_cnt = None
+        self.seed = None
+        self.numberOfSlides = None
+        self.frameCounter = None
```

Comment:
@mrshu I don't want to be disrespectful but `self.saved_cnt` is your piece of code =)